LibreHardwareMonitorLib/Hardware/Computer.cs

```diff
@@ -166,11 +167,13 @@ public bool IsGpuEnabled
                     {
                         Add(new AmdGpuGroup(_settings));
                         Add(new NvidiaGroup(_settings));
+                        Add(new IntelGpuGroup(_groups.SelectMany(x => x.Hardware).Select(x => x as CPU.IntelCpu).FirstOrDefault(x => x != null), _settings));
```

Comment:
Would this be possible to solve differently, for example by requesting this energy multiplier in the iGPU class itself? It's not necessarily guaranteed that GPU is enabled before CPU.

Reply:
Possible, but alot of code in InteCpu, GenericCpu and CpuGroup that is used to calculate that, and I didn't want to duplicate that. Would it make more sense to make sure we have a IntelCpu instance (event creating one on the fly even if that is disabled) ??

Reply:
Yeah, I think it should be OK to create a CpuGroup instance (but not add it) in that case.